### DISCUSSION TITLE
prConcurrentLimit did not work.  I set it to 3, but 7 PRs were opened

### DISCUSSION THREAD

**Discussion by gschueler** [renovatebot/renovate#15947] [Request Help]
I'm new to the renovate configuration, but in the "welcome" PR, I modified the config to add `"prConcurrentLimit":3`.  However, when I merged the PR, 7 new PRs were opened.  I thought that setting would limit it to 3.  Is that incorrect?  

Here is the welcome PR https://github.com/rundeck/rundeck/pull/7683

List of renovate PRs https://github.com/rundeck/rundeck/pulls/app%2Frenovate

**Comment by rarkins** [comment]
PR concurrency limits are not applied to security fixes. Normally that's desirable..

**Reply by gschueler** [reply]
ok.  Perhaps the docs should mention that.  It just says it states it applies a maximum concurrent limit.

https://docs.renovatebot.com/configuration-options/#prconcurrentlimit

**Reply by rarkins** [reply]
PRs welcome - docs content is all in this repo

**Reply by gschueler** [reply]
I sent a PR here: https://github.com/renovatebot/renovate/pull/15950
